### PR TITLE
packages upgrades

### PR DESCRIPTION
**Pull request by dvdknaap**
bootstrap 4 support and packages upgraded to latest version

### CODE REVIEW

**Comment by dvdknaap** [comment]
I guess this project died out if you want to use this upgrade just use my fork

Change your package.json to:
`        "bootstrap-material-design": "git+https://github.com/dvdknaap/bootstrap-material-design.git#master",`

**Comment by FezVrasta** [comment]
No the project isn't dead because I haven't replied to your PR within 4 hours during weekend while I baby sit a new puppy 🙂

**Comment by dvdknaap** [comment]
> No the project isn't dead because I haven't replied to your PR within 4 hours during weekend while I baby sit a new puppy

No because i saw that some issue where unanswered for some time, i'm sorry glad it's still active :+1:

File: scss/mixins/_forms.scss [hunk 1]
```diff
@@ -156,7 +156,7 @@
     // on focus set borders and labels to the validation color
 
     // Use the BS provided mixin for the bulk of the color
-    @include form-validation-state("valid", $label-color);
+    @include form-validation-state("valid", $label-color, "none");
```
Comment (FezVrasta) [inline scss/mixins/_forms.scss:159]: May you help me understand this change please?

Comment (dvdknaap) [reply scss/mixins/_forms.scss:159]: This update is required because bootstrap 4 expects extra param

**Pull request merged**